components/src/dynamo/common/utils/tests/test_nvtx_utils.py

```
@@ -0,0 +1,466 @@
+        handle = len(self.started)
+        self.started.append(attr)
+        return handle
+
+    def end_range(self, handle):
+        self.ended.append(handle)
+
+
+class _StubAnnotate:
+    """Stand-in for ``nvtx.annotate``: decorator and context manager."""
+
+    instances: list = []
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)